lib/clean.js

```diff
@@ -1,15 +1,17 @@
 'use strict'
 
-const rm = require('rimraf')
-const log = require('npmlog')
+const fs = require('fs')
+const log = require('./log')
 
-function clean (gyp, argv, callback) {
+async function clean (gyp, argv) {
   // Remove the 'build' dir
-  var buildDir = 'build'
+  const buildDir = 'build'
 
   log.verbose('clean', 'removing "%s" directory', buildDir)
-  rm(buildDir, callback)
+  await fs.promises.rm(buildDir, { recursive: true, force: true })
```

Comment:
`require('fs/promises')` would help that

Reply:
This has been updated to `fs/promises`.